data_broker.js

```diff
@@ -190,6 +199,8 @@ class DataBroker extends EventTarget {
       this._header = parseBamHeaderData(headerText);
       this.emitEvent('header', this._header);
 
+      this.emitEvent('bamview-data-request-end', null)
```

Comment:
Rather than having 3 events for `read-depth`, `header`, and `bamview-data-request-end`, let's just replace them with a single event called `alignment-data` which emits this:

```javascript
{
  header: this._header,
  readDepthData: this._readDepthData,
}
```